src/meta/topics/dependencies.md

```diff
@@ -0,0 +1,90 @@
+# Dependencies
+
+There are several dependencies that Node.js relies on to work the way it does.
+
+- [Libraries](#Libraries)
+	- [V8](#V8)
+	- [libuv](#libuv)
+	- [http-parser](#http-parser)
+	- [c-ares](#c-ares)
+	- [OpenSSL](#OpenSSL)
+	- [zlib](#zlib)
+- [Tools](#Tools)
+	- [npm](#npm)
+	- [gyp](#gyp)
+	- [gtest](#gtest)
```

Comment:
I generally distrust the tab character, since it gets interpreted different ways by different people, leading to misaligned whitespace for a subset of viewers.

I feel strongly about consistency — that we at least attempt to have consistent formatting. Therefore this tends to be an argument against tab characters :)